serocalculator.Rproj

```diff
@@ -18,6 +18,6 @@ StripTrailingWhitespace: Yes
 BuildType: Package
 PackageUseDevtools: Yes
 PackageInstallArgs: --no-multiarch --with-keep.source
-PackageRoxygenize: rd,collate,namespace
+PackageRoxygenize: rd,collate,namespace,vignette
```

Comment:
this addition seems reasonable; what exactly does it do?

Reply:
It enables the use of Roxygen to generate vignettes. This setting is activated when the vignette option is selected in the Roxygen settings under Project Options. I selected it when the vignettes were not building locally.

<img width="372" alt="image" src="https://github.com/UCD-SERG/serocalculator/assets/731279/c09aa1a2-90df-4a80-8754-de40b7c7cfa2">

Reply:
Ah, good to know. I've never understood what that option does, so I did some digging today, and discovered that it changes what happens when you press the `document()` button in the Build tab:

without checkbox:

`devtools::document(roclets = c('rd', 'collate', 'namespace'))`

with checkbox:

`devtools::document(roclets = c('rd', 'collate', 'namespace', 'vignette'))`

(more details: https://rpubs.com/jennybc/build-a-basic-package)

Unfortunately, even with this option checked, ["Articles"](https://r-pkgs.org/vignettes.html#sec-vignettes-article) in the `vignettes/articles` subdirectory do not appear to get run. That's why you keep running into errors on GitHub that don't come up on your computer.

I'm not sure how to automatically check articles on your local computer; there's an old issue on the `devtools` GitHub page (https://github.com/r-lib/devtools/issues/1886), but it looks like a dead end. @chrisorwa, could you spend an hour looking for alternative solutions? It would be great to have a better system for checking articles.